CIScripts/Test/TestRunner.ps1

```diff
@@ -46,29 +46,24 @@ function Invoke-TestScenarios {
         "vRouterAgentService.Tests.ps1"
     )
 
-    $TotalFailedCount = 0
-
     $TestPaths = Get-ChildItem -Recurse -Filter "*.Tests.ps1"
-    Foreach ($TestPath in $TestPaths) {
-        if ($TestPath.Name -in $TestsBlacklist) {
-            Write-Host "Skipping $($TestPath.Name)"
-            continue
-        }
-
-        $PesterRunScript = @{
-            Path=$TestPath.FullName; 
+    $WhitelistedTestPaths = $TestPaths | Where-Object { !($_.Name -in $TestsBlacklist) }
+    $PesterScripts = $WhitelistedTestPaths | ForEach-Object {
+        @{
+            Path=$_.FullName;
             Parameters= @{
                 TestbedAddr=$Sessions[0].ComputerName;
                 ConfigFile=$TestConfigurationFile
             }; 
             Arguments=@()
         }
-
-        $Results = Invoke-Pester -PassThru -Script $PesterRunScript
-        $TotalFailedCount += $Results.FailedCount
     }
-    Write-Host "Num failed tests: $TotalFailedCount"
-    if ($TotalFailedCount -gt 0) {
+    $OutputXmlPath = "$PSScriptRoot\testReport.xml"
```

Comment:
`CIScripts/Test/testReport.xml` is a bad output path. We're mixing test results with scripts.

Let's put it somewhere in the root, outside of contrail-windows-ci directory altogether.

(this issue exists AFAIK for build artifacts and project repositories - they should never be stored/outputted inside CIScripts/ or contrail-windows-ci. But it's hard to decouple them all at the same time. So for now, let's not contribute to the mess further.)

How I see it for the nearest future:

```
contrail-windows-ci/{CIScripts,ansible,...}
bin/{...}
src/{controller,vrouter......}
testReport.xml <-- not sure exactly about this, but I don't think this should be outputted under contrail-windows-ci dir.
```